analysis/topology/topological_space.lean

```diff
@@ -635,6 +645,70 @@ lemma tendsto_nhds_unique [t2_space α] {f : β → α} {l : filter β} {a b : 
   (hl : l ≠ ⊥) (ha : tendsto f l (nhds a)) (hb : tendsto f l (nhds b)) : a = b :=
 eq_of_nhds_neq_bot $ neq_bot_of_le_neq_bot (map_ne_bot hl) $ le_inf ha hb
 
+def separated_sets (s t : set α) : Prop :=
+∃u v : set α, is_open u ∧ is_open v ∧ s ⊆ u ∧ t ⊆ v ∧ u ∩ v = ∅
+
+lemma separated_sets.symm {s t : set α} : separated_sets s t → separated_sets t s :=
+assume ⟨u, v, uo, vo, su, tv, uv⟩, ⟨v, u, vo, uo, tv, su, inter_comm u v ▸ uv⟩
+
+lemma separated_sets.comm {s t : set α} : separated_sets s t ↔ separated_sets t s :=
+iff.intro separated_sets.symm separated_sets.symm
+
+lemma separated_compact_of_separated_pt {s : set α} :
+  (∀x, x ∉ s → separated_sets {x} s) →
+  (∀k, compact k → k ∩ s = ∅ → separated_sets k s) :=
+assume hsep k k_cpt ks,
```

Comment:
Why the `lemma` followed directly by `assume`? Give the names to the parameters of your lemma:
```lean
lemma separated_compact_of_separated_pt {s : set α}
  (hsep : ∀x, x ∉ s → separated_sets {x} s) (k) (k_cpt : compact k) (ks : k ∩ s = ∅) :
  separated_sets k s :=
```
Also I would change the name to `separated_sets_of_compact` (or something like this)